EventsExpress.Core/Services/PhotoService.cs

```diff
@@ -51,6 +53,55 @@ public async Task<Photo> AddPhoto(IFormFile uploadedFile)
             return photo;
         }
 
+        public async void AddAvatar(Guid userId, string url)
+        {
+            var user = _db.UserRepository.Get("Photo").FirstOrDefault(u => u.Id == userId);
+            user.Photo = await AddPhotoByURL(url);
+            _db.UserRepository.Update(user);
+            await _db.SaveAsync();
+        }
+
+        public async Task<Photo> AddPhotoByURL(string url)
+        {
+            if (!IsImageUrl(url))
+            {
+                throw new ArgumentException();
+            }
+
+            var webClient = new WebClient();
```

Comment:
Please use `HttpClient` instead of `WebClient` or `WebRequest`. Read how to inject `HttpClient` in microsoft docs.